src/app/theme/components/progressBarRound/progressBarRound.directive.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Created by n.poltoratsky
+ * on 28.06.2016.
+ */
+(function () {
+    'use strict';
+
+    angular.module('BlurAdmin.theme.components')
+        .directive('progressBarRound', progressBarRound);
+
+    /** @ngInject */
+    function progressBarRound(baProgressModal) {
+        return {
+            restrict: 'E',
+            templateUrl: 'app/theme/components/progressBarRound/progressBarRound.html',
+            link:function($scope, element, attrs) {
+                $scope.baProgressDialog = baProgressModal;
+                $scope.$watch(function () {
+                    return baProgressModal.getProgress();
+                }, animateBar);
+
+                function animateBar() {
+                    var circle = document.getElementById('loader');
```

Comment:
Using IDs is not right decision here. Despite your component is singletone, user of framework can define some other element with id `loader` and your code will break